libs/image/include/image/KtxBundle.h

```diff
@@ -123,6 +123,25 @@ class KtxBundle {
      */
     bool setBlob(KtxBlobIndex index, uint8_t const* data, uint32_t size);
 
+    // The following constants help clients populate the "info" struct. Most of them have corollary
+    // constants in the OpenGL headers.
+
+    static constexpr uint32_t RED = 0x1903;
```

Comment:
Should those be defined in enums?

Reply:
I actually prefer static consts for these since the spec says the header is uint32_t and it is a fairly general container format.